lib/ts/recipe/session/sessionClass.ts

```diff
@@ -18,6 +18,16 @@ import STError from "./error";
 import { SessionClaim, SessionClaimValidator, SessionContainerInterface, TokenTransferMethod } from "./types";
 import { Helpers } from "./recipeImplementation";
```

Comment:
We should remove updateAccessTokenPayload since this is a breaking change anyway, and we do not want people to call updateAccessTokenPayload thinking that it's a merge.